core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -564,6 +568,20 @@ def _fit(self,
         X = self.preprocess(X)
         self.model = self.model.fit(X, y)
 
+    def _apply_temperature_scaling(self, y_pred_proba):
+        if self.temperature_scalar is not None and self.problem_type in PROBLEM_TYPES_CLASSIFICATION:
+            if self.problem_type == BINARY:
```

Comment:
Add `# TODO: This is expensive to convert at inference time, try to avoid in future`

Reply:
added